docs/integrations/sources/surveymonkey.md

```diff
@@ -1,48 +1,64 @@
 # SurveyMonkey
 
-This page guides you through the process of setting up the SurveyMonkey source connector.
+Learn how to set up the SurveyMonkey source connector. A source connector extracts data from SurveyMonkey and transfers the data to a destination. Learn more about connectors [here](/using-airbyte/core-concepts/).
 
 :::note
 
-OAuth for Survey Monkey is officially supported only for the US. We are testing how to enable it in the EU at the moment. If you run into any issues, please [reach out to us](mailto:product@airbyte.io) so we can promptly assist you.
+OAuth for SurveyMonkey is available in the US only. We are testing how to enable OAuth in the EU. [Reach out to us](mailto:product@airbyte.io) if you run into any issues with authentication.
 
 :::
 
 <!-- env:oss -->
-## Prerequisites
+## Know before you begin
 
-**For Airbyte Open Source:**
-
-* Access Token
+You need a SurveyMonkey access token if you're using Airbyte Open Source to set up the connector. Use this token to authenticate your SurveyMonkey account with Airbyte Open Source.
```

Comment:
1. We never include a list with just one bullet point. Work that into a paragraph.

2. From what I understand in the next section, I have two ways to set up SurveyMonkey in Airbyte. If I'm using Airbyte Cloud, I don't need an access token; if I'm using Open Source, I do. In this case, we need to clearly call out the dependency so Airbyte Cloud users don't waste their time.

3. If the goal of this prereq is to get a SurveyMonkey access token, then this isn't a prereq, then this is something we're doing in step 1. If this is all true, I recommend changing this section to "Know before you begin."